assets/queries/cloudFormation/aws_lambda_functions_with_full_privileges/query.rego

```diff
@@ -0,0 +1,47 @@
+package Cx
+
+CxPolicy [ result ] {
+	resources := input.document[i].Resources
+ 	resource := resources[name]
+    
+  resource.Type == "AWS::Lambda::Function"
+    
+  # check if the role referenced in the lambda function properties is defined in the same template
+  fullRole := split(resource.Properties.Role,".")
+  role := fullRole[0]
+  resources[role]
+	
+	checkPolicies(resources[role].Properties.Policies)
+  
+	result := {
+              "documentId": 	  	input.document[i].id,
+              "searchKey": 	      sprintf("Resources.%s.Properties.Policies.PolicyDocument", [role]),
+              "issueType":		    "IncorrectValue",  
+              "keyExpectedValue": sprintf("Resources.%s.Properties.Policies.PolicyDocument is not giving admin privileges to Resources.%s ", [role,name]),
+              "keyActualValue": 	sprintf("Resources.%s.Properties.Policies.PolicyDocument is giving admin privileges to Resources.%s ", [role,name])
+              }
+}
+
+checkPolicies(policies) {
+	some j,k
+   		statement := policies[j].PolicyDocument.Statement[k]
+    	statement.Effect == "Allow"
+    checkResource(statement)
+    checkAction(statement)
+}
+
+checkResource(statement) {
+	statement.Resource == "*"
+}
+
+checkResource(statement) {
+	endswith(statement.Resource,"*")
+}
+
+checkAction(statement) {
+	statement.Action[_] == "*"
+}
+	
+checkAction(statement) {
+	endswith(statement.Action[_],"*")
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline